content/docs/gto/command-reference/annotate.md

```diff
@@ -44,6 +46,11 @@ awesome-model:
 
 This information can be later retrieved by running `gto describe` command.
 
+`--custom` option allows you to add custom metadata. Within `gto annotate` GTO
+treats input as string, but you can edit artifacts.yaml to add arbitrary
+information that fits yaml format, such as dict or list, or more complex
+structures.
```

Comment:
Doesn't seem important enough to be the only option explained directly in the Description. Probably adding to the Options section is enough?